src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -158,6 +182,25 @@
  * </li>
  * </ul>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * import static java.lang.System.out; // Violation; it should be 'sorted'
+ * import static java.lang.Math.*; // OK
+ * import java.io.IOException; // Violation; Groups aren't separated by blank line
+ *
+ * import java.net.URL;
+ * import java.security.KeyManagementException;
+ *
+ * import javax.net.ssl.TrustManager;
+ *
+ * import javax.net.ssl.X509TrustManager; // Violation; Groups should not separate internally
+ *
+ * import org.apache.http.conn.ClientConnectionManager;
+ *
+ * public class SomeClass { ... }
+ * </pre>
+ * <p>
```

Comment:
Please move this example to line 226, below the configuration. Take a look at the [site](https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/ImportOrderBranch_2021161402/config_imports.html#ImportOrder) generated by this: 
![image](https://user-images.githubusercontent.com/31252532/104107398-d6a3fa80-5289-11eb-8d88-35793cb3ff53.png)

The code example should be below the check configuration.

Reply:
done

Reply:
Resolved